src/linearmixedmodel.jl

```diff
@@ -359,12 +359,22 @@ function condVartables(m::MixedModel{T}) where {T}
     return NamedTuple{fnames(m)}((map(_cvtbl, condVar(m), m.reterms)...,))
 end
 
+function StatsBase.confint(m::MixedModel{T}; level=0.95) where {T}
+    cutoff = sqrt.(quantile(Chisq(1), level))
+    β, std = m.β, m.stderror
+    return DictTable(;
+        coef=coefnames(m),
+        lower=β .- cutoff .* std,
+        upper=β .+ cutoff .* std
```

Comment:
what happens in rank deficiency?

Reply:
I'm not sure.  Part of me just wants to tell users to ensure that their model matrices have full rank but that might be an unpopular suggestion.  I'll have to look up what happens with the `stderror` property in that case.